bin/all_sky_search/pycbc_fit_sngls_over_multiparam

```diff
@@ -459,38 +461,49 @@ elif numpy.isfinite(_smooth_cut[args.smoothing_method]):
     # Sort the values to be smoothed by parameter value
     logging.info("Smoothing ...")
     slices = [slice(l,r) for l, r in zip(lefts, rights)]
+    nabove_sort =  nabove[par_sort]
+    invalphan_sort = invalphan[par_sort]
+    ntotal_sort = ntotal[par_sort]
```

Comment:
This is the dominant saving cost - this was being done within the loop, meaning an $O(n^2)$ operation rather than $O(n)$